public/app/features/dashboard-scene/scene/ScopesScene.tsx

```diff
@@ -0,0 +1,133 @@
+import { css, cx } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { SceneComponentProps, sceneGraph, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { IconButton, useStyles2 } from '@grafana/ui';
+
+import { ScopesDashboardsScene } from './ScopesDashboardsScene';
+import { ScopesFiltersScene } from './ScopesFiltersScene';
+
+export interface ScopesSceneState extends SceneObjectState {
+  dashboards: ScopesDashboardsScene;
+  filters: ScopesFiltersScene;
+  isExpanded: boolean;
+  isViewing: boolean;
+}
+
+export class ScopesScene extends SceneObjectBase<ScopesSceneState> {
+  static Component = ScopesSceneRenderer;
+
+  constructor() {
+    super({
+      dashboards: new ScopesDashboardsScene(),
+      filters: new ScopesFiltersScene(),
+      isExpanded: false,
+      isViewing: false,
+    });
+
+    this.addActivationHandler(() => {
+      this.state.filters.fetchScopes();
+
+      const filtersValueSubscription = this.state.filters.subscribeToState((newState, prevState) => {
+        if (newState.value !== prevState.value) {
+          this.state.dashboards.fetchDashboards(newState.value);
```

Comment:
This is eagerly loading dashboards even when the dashboards list object ain't rendered. We can consider optimising this in the future. Would probably require inversing the reposibility and fetching the dashboard on ScopesDashboardsScene activation and subscribing to the scope selector from within that object.

However, let's consider this an improvement for the followup PR as this is not blocking this work.

Reply:
Yup! Let's address this in a subsequent PR.